spec/hardware/gpu.fmf

```diff
@@ -0,0 +1,15 @@
+summary:
+    Choose system according to the available graphics processing unit (gpu)
+example:
+  - |
+    # GPU-related stuff grouped together.
+    # Can be mapped from the output of `lshw -C display` command output.
+    gpu:
+        # GPU product name.
+        product-name: G86 [Quadro NVS 290]
```

Comment:
Wouldn't `product-name` be the same field as `device-name` from the device specification PR, https://github.com/teemtee/tmt/pull/1759/files#diff-9dd87f09c4ab902df670b30e00fe89d0966a3499985258b1d1f731e52f9fd322R12?

Reply:
@happz looks like it, added as a discussion item.

Reply:
@happz edited, please re-review